src/Controls/src/Core/Handlers/Items/iOS/TemplatedCell.cs

```diff
@@ -172,6 +172,8 @@ void SetRenderer(IPlatformViewHandler renderer)
 
 			InitializeContentConstraints(platformView);
 
+			UpdateVisualStates();
```

Comment:
Fix CollectionView not applying "Selected" VisualState to pre-selected items on iOS
https://github.com/xamarin/Xamarin.Forms/pull/14672/files